Dockerfile

```diff
@@ -0,0 +1,105 @@
+# To use or update to a ruby version, change {BASE_RUBY_IMAGE}
+ARG BASE_RUBY_IMAGE=ruby:2.7.1-alpine
+# BASE_RUBY_IMAGE_WITH_GEMS_AND_NODE_MODULES will default to early-careers-framework-gems-node-modules
+# building all layers above it if a value is not specidied during the build
+ARG BASE_RUBY_IMAGE_WITH_GEMS_AND_NODE_MODULES=early-careers-framework-gems-node-modules
+
+# Stage 1: install-gems-node-modules, download gems and node modules.
+FROM ${BASE_RUBY_IMAGE} AS install-gems-node-modules
+
+ARG BUILD_DEPS="git gcc libc-dev make nodejs yarn postgresql-dev build-base libxml2-dev libxslt-dev ttf-ubuntu-font-family"
+ENV WKHTMLTOPDF_GEM=wkhtmltopdf-binary-edge-alpine
```

Comment:
What's this for?

Reply:
Good spot, apparently generating pdfs, which we probably don't need any time soon.

There are a few other environment variables that we don't use (notify ones), but I am inclined to keep them, since we will almost certainly want them.